packages/perseus/src/version.js

```diff
@@ -1,7 +1,7 @@
 // @flow
 export default {
     apiVersion: {
-        major: 11,
+        major: 12,
         minor: 0,
     },
```

Comment:
Let's give this a try for now.  We can look at removing `apiVersion` in the future once we figure out how to update the code that currently references it in webapp.